_pages/HeatherGranger.txt

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: HeatherGranger
+pageUrl: https://heathergranger.github.io/ 
+timestamp: 2017-01-23
+---
```

Comment:
@HeatherGranger change the `.txt` extension to `.yaml`, and then remove the `http://` and the last `/`, and you should be set!